component/toast/toast.css

```diff
@@ -0,0 +1,43 @@
+.component_box.toast_box {
+    height: 10rem;
+}
+
+.container {
+    width: 25rem;
+    position: relative;
+    bottom: 1.5rem;
+}
+
+#SnackBar {
```

Comment:
unless really necessary, try to use classes instead of ids ac much as you can

Reply:
ok i will change this